### PR TITLE
Add Vector to projects using Pest

### PR DESCRIPTION
**Pull request by lucperkins**
This PR adds [Vector](https://vector.dev) to the list of projects using Pest. We're using Pest to power [Timber Remap Language](https://vector.dev/docs/reference/remap), a lean data transformation language for observability event data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,8 +163,10 @@ Digit:   2
 * [ui_gen](https://github.com/emoon/ui_gen)
 * [ukhasnet-parser](https://github.com/adamgreig/ukhasnet-parser)
 * [ZoKrates](https://github.com/ZoKrates/ZoKrates)
+* [Vector](https://github.com/timberio/vector)
 
 ## Special thanks
 
 A special round of applause goes to prof. Marius Minea for his guidance and all
 pest contributors, some of which being none other than my friends.
+
```
Comment (MarinPostma) [inline README.md:172]: Can you remove this empty line please?

**Review by MarinPostma — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucperkins) [reply README.md]: Happy to do so. In the future, however, I'd recommend relaxing these kinds of low-value requirements.

Comment (MarinPostma) [reply README.md]: Thanks :)

**Review by MarinPostma — APPROVED** [review]

**Comment by MarinPostma** [comment]
Thanks for your contribution! 


bors r+

**Comment by CAD97** [comment]
Wait, is this PR crashing as well

bors: r=MarinPostma

**Comment by CAD97** [comment]
cc @notriddle

This PR is crashing bors as well with the same permission error, is much smaller, and #511 merged fine...

**Comment by CAD97** [comment]
There's a permissions issue blocking bors currently; merging manually.

**Pull request merged**